packages/pancake-uikit/src/widgets/Menu/components/Accordion.tsx

```diff
@@ -69,7 +69,21 @@ const Accordion: React.FC<Props> = ({
   );
 };
 
-export default React.memo(
-  Accordion,
-  (prev, next) => prev.isPushed === next.isPushed && prev.isActive === next.isActive
-);
+export default React.memo(Accordion, (prev, next) => {
+  const prevChildren = prev.children;
+  const nextChildren = next.children;
+
+  const isChildrenSame =
+    Array.isArray(prevChildren) && Array.isArray(nextChildren)
+      ? prevChildren.every((value, index) => {
+          const prevMenuEntry = value as ReactElement;
+          const nextMenuEntry = nextChildren[index] as ReactElement;
+          return (
+            prevMenuEntry.props.isActive === nextMenuEntry.props.isActive &&
+            prevMenuEntry.props.isPushed === nextMenuEntry.props.isPushed
+          );
+        })
+      : true;
+
+  return prev.isPushed === next.isPushed && prev.isActive === next.isActive && isChildrenSame;
+});
```

Comment:
I don't think this is the right solution for this problem. It seems that we are trying to solve a problem that doesn't exist. In my opinion we should remove `React.memo`. The Accordion is not a complex and re-renders will not cause a performance issue. React is pretty good at optimizing.

@RabbitDoge What do you think?

Reply:
I would rather remove the memo instead of adding some complex and hacky code , for sure. This logic could as well remove the small gain we have from the memo

Reply:
@memoyil Can you revert this change and simply remove the memo?

Reply:
done